src/shared/bg_pmove.cpp

```diff
@@ -2418,37 +2418,33 @@ PM_CorrectAllSolid
 */
 static int PM_CorrectAllSolid( trace_t *trace )
 {
-	int    i, j, k;
-	vec3_t point;
-
 	if ( pm->debugLevel > 1 )
 	{
 		Log::Notice( "%i:allsolid\n", c_pmove );
 	}
 
 	// jitter around
-	for ( i = -1; i <= 1; i++ )
+	vec3_t point;
```

Comment:
Understandable, but considering this is done a lot, I wanted to reduce the number of calculations. This commit have zero impact on the API though, and thus is pretty easy to remove from the stack. I also don't think I touched that area of code later, so that probably means no conflict if you prefer it to be removed.

Reply:
Moving the code that sets `point` around won't make the code more efficient, and this is the part that is hard to read, so I'd rather have this change dropped.

Removing the extra copy into pml.groundTrace is good.

Reply:
fixed.